source/manual/howto-switch-default-branch.html.md

```diff
@@ -0,0 +1,26 @@
+---
+owner_slack: "#govuk-developers"
+title: How to switch a git default branch from master to main
+section: GitHub
+layout: manual_layout
+parent: "/manual.html"
+---
+
+These steps are for switching the default branch of a repo to `main`.
+
+1. Rename the `master` branch in GitHub to `main` for that repo. See [instructions from Github for renaming an existing default branch](https://github.com/github/renaming#renaming-existing-branches).
+1. Update the [Release app](https://release.publishing.service.gov.uk/applications/):
+  - Go to the app entry from the [listing](https://release.publishing.service.gov.uk/applications/).
+  - Hit the `Edit` tab.
+  - Switch under the dropdown for `GitHub repository default branch`.
+1. [Reset the Jenkins build number](https://docs.publishing.service.gov.uk/manual/test-and-build-a-project-on-jenkins-ci.html#fixing-the-build-number).
+1. Delete the master branch locally, otherwise you might rebase the wrong branch by mistake!
+
+## Additional things to check
+
+- Update references in Jenkins to pulling down the `master` branch of the app.
```

Comment:
It might be worth adding some instructions on how to use the GitHub search feature to find these. (https://github.com/alphagov/govuk-puppet/search?l=HTML%2BERB&q=master)

Reply:
Nice! Done